### PR TITLE
fixed some xcode 10 warnings and ui changes

### PR DESCRIPTION
**Pull request by AAAstorga**
#78 
#77 
#76

### CODE REVIEW

File: Cornell Sun/Cornell Sun/Cells/VideoCell.swift [hunk 1]
```diff
@@ -35,9 +35,11 @@ final class VideoCell: UICollectionViewCell {
     }
 
     func setupViews() {
+        backgroundColor = .white
         addSubview(videoWebView)
         videoWebView.snp.makeConstraints { make in
-            make.width.height.equalToSuperview()
+            make.height.equalToSuperview()
+            make.leading.trailing.equalToSuperview().inset(16)
```
Comment (mindylou) [inline Cornell Sun/Cornell Sun/Cells/VideoCell.swift]: make this inset a constant

File: Cornell Sun/Cornell Sun/Settings Section/ContactViewController.swift [hunk 2]
```diff
@@ -239,7 +239,7 @@ class ContactViewController: UIViewController, UITextFieldDelegate, MFMailCompos
     }
     
     func getText() -> String {
-        switch type {
+        switch type! {
```
Comment (mindylou) [inline Cornell Sun/Cornell Sun/Settings Section/ContactViewController.swift]: why is this force cast? is there a way to safely unwrap this

File: Cornell Sun/Cornell Sun/Settings Section/ContactViewController.swift [hunk 3]
```diff
@@ -250,7 +250,7 @@ class ContactViewController: UIViewController, UITextFieldDelegate, MFMailCompos
     }
     
     func getHeader() -> String {
-        switch type {
+        switch type! {
```
Comment (mindylou) [inline Cornell Sun/Cornell Sun/Settings Section/ContactViewController.swift]: same here - and all of the other ones

File: Cornell Sun/Cornell Sun/Settings Section/ContactViewController.swift (hunk 2 above)
Comment (AAAstorga) [reply Cornell Sun/Cornell Sun/Settings Section/ContactViewController.swift]: idk but xcode tells me they are optional,... but they are like not optional lol

Comment (mindylou) [reply Cornell Sun/Cornell Sun/Settings Section/ContactViewController.swift]: lol that's so weird

**Pull request merged**